uv.gyp

```diff
@@ -274,19 +274,44 @@
           },
         }],
         [ 'OS=="aix"', {
-          'sources': [ 'src/unix/aix.c' ],
+          'variables': {
+            'TrueOS': '<!(uname -s)',
```

Comment:
Suggestion: name this `uname_s` or `os_name` or something?

Reply:
I think os_name is better.

Reply:
Updated the code.